### PR TITLE
Add Copy item ID command

### LINKED ISSUE #21: Feature: Get resource URL
I think it'd be useful to have an option to just grab the URL of the currently selected resource. This could be used if you want to share a resource, save it for other uses or browse it in a non default browser.

### PR DESCRIPTION
**Pull request by stuartleeks**
Fixes #21 

* Enables commands without a keybinding
* Adds command to copy current item ID

### CODE REVIEW

**Comment by lawrencegripper** [comment]
Ps. Command pallet is very very cool now. Like the idea of adding in commands without keyboard shortcuts.

**Comment by stuartleeks** [comment]
@jjcollinge - @lawrencegripper is right that we will have a clash between our PRs. I've moved the clipboard code to a shared method (i.e. encapsulated the wsl vs non-WSL test etc inside a single function) and reused this in another command. Since your PR is marked as draft, are you ok to merge this PR and then fix up in yours?

**Comment by jjcollinge** [comment]
Yeh - I'll fix up once this is merged

**Comment by stuartleeks** [comment]
@jjcollinge - thank you! Are you ok to look at this and approve if you're happy? Then you can update your PR and get that merged (I'm looking forward to that :-))

File: internal/pkg/keybindings/listhandlers.go [hunk 1]
```diff
@@ -722,3 +722,48 @@ func (h *CommandPanelAzureSearchQueryHandler) CommandPanelNotification(state vie
...
+}
+
+func (h ListCopyItemIDHandler) Fn() func(g *gocui.Gui, v *gocui.View) error {
+	return func(g *gocui.Gui, v *gocui.View) error {
+		return h.Invoke()
+	}
+}
+
+func (h *ListCopyItemIDHandler) DisplayText() string {
+	return "Copy current item ID"
```
Comment (jjcollinge) [inline internal/pkg/keybindings/listhandlers.go]: nit: I think "Copy current resource ID" is more consistent with Azure terminology.

**Comment by stuartleeks** [comment]
ah, good spot!

**Comment by stuartleeks** [comment]
Ok, rebased on previous merge and fixed up the other messages to use "resource ID"

**Comment by stuartleeks** [comment]
All done - over to you for #166 :-)

**Pull request merged**